### ISSUE TITLE
Text under Create a new Workbook are not having 4.5:1 color contrast ratio in Create a new Workbook page.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#273]
### Steps to Reproduce

1. Launch Xamrin Workbook application. 
2.In the "Create a new workbook" window that opens, observe the Colour Contrast ratio for text under "Create a new Workbook"

### Expected Behaviour
Colour contrast ratio should be minimum 4.5:1

### Actual Behaviour
Colour contrast ratio is lesser than 4.5:1 (i.e., 3.9:1)
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File

![issue 2](https://user-images.githubusercontent.com/22956130/39860843-50c0e948-542e-11e8-831a-95a79285e8e5.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1828
